### ISSUE TITLE
Anti-Decompiler / Fail on decompile crashing

### ISSUE THREAD

**Issue by juniorjacob** [LoukaMB/SynapseX#2]
Hello, I have a anti-decompiler that still works, I would like it fixed and/or it to fail when it freezes the Java executable.

Heres how it works and the code for it:

local a = not 1 or not 1 or not 1 or not 1 or not 1 or not 1 or not 1 or not 1 not 1 or not 1 or not 1 or not 1 not 1 or not 1 or not 1 or not 1 not 1 or not 1 or not 1 or not 1 not 1 or not 1 or not 1 or not 1 not 1 or not 1 or not 1 or not 1 not 1 or not 1 or not 1 or not 1 not 1 or not 1 or not 1 or not 1

Basically the decompiler has to check through all the or's and that when spammed a 100k times will crash the game, depending on pc. It also doesn't affect performance for the actusl script, oddly enough, so its a great way to just crash the decompiler, and ROBLOX.

**Comment by juniorjacob** [comment]
For a good test make sure you spam the or statement for 1 local.

**Comment by DocorDerank** [comment]
Wouldn't this just show as `local a = false` under the decompiler?

**Comment by juniorjacob** [comment]
it will if the decompiler doesnt crash, if you do it long enough it goes through each or before making it false
